src/snapshot_creator.cc

```diff
@@ -37,7 +37,7 @@ v8::StartupData MakeSnapshot(const char* js_filename, const char* js_source) {
   // Note that using kKeep here will cause segfaults. This is demoed in the
   // "SnapshotBug" test case.
```

Comment:
Should remove this comment. But let's do it in a future commit - I dont want to wait for CI again